ts/session/utils/Protobuf.ts

```diff
@@ -9,6 +9,12 @@ import ByteBuffer from 'bytebuffer';
 export function convertToTS(object: any): any {
   // No idea why js `ByteBuffer` and ts `ByteBuffer` differ ...
   if (
```

Comment:
`object instanceof Uint8Array` doesn't work here?

The reason I had done it for `ByteBuffer` below is that `ByteBuffer` in js and `ByteBuffer` in ts are not equivalent.

Reply:
yes it is working with `instanceof`